### PR TITLE
Add emoji to readme in webinar

### PR DESCRIPTION
**Pull request by githubteacher**
## Emoji in README.md

@brianamarie What do you think?

### CODE REVIEW

**Review by LizzyKemp — COMMENTED** [review]
Love the Emoji's

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # GitHub Teacher's Blog
 
 ### What You'll Find Here
-This is where GitHubTeacher puts thoughts and experiments using GitHub Pages and Jekyll. Please feel free to open any pull requests or ask any questions about anything you see here! 
+This is where GitHubTeacher puts thoughts and experiments using GitHub Pages and Jekyll. Please feel free to open any pull requests or ask any questions about anything you see here! :tada: :sparkles: :pizza:
```
Comment (brianamarie) [inline README.md:4]: @githubteacher I think this looks great!

**Review by brianamarie — APPROVED** [review]
👍

File: README.md (hunk 1 above)
Comment (KentBarr) [reply README.md:4]: Umm, pizza.

**Pull request merged**